My Simple App/Business Logic/AddressStore/AddressObject.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  AddressObject.swift
+//  My Simple App
+//
+//  Created by Erick Sanchez on 8/12/20.
+//  Copyright © 2020 Erick Sanchez. All rights reserved.
+//
+
+import Foundation
+
+struct AddressObject: Codable {
```

Comment:
woodward4422 2 days ago
I see the convention of calling the codable object ModelDTO in this case it would be AdressDTO this means Data to Object as its clear as this is the object that is not what is consumed/used on the client and instead what is being decoded

LinnierGames 1 hour ago Author Owner
😯 neat! I haven't come across this convention at Google, yet. But, that might be because I've only dug so far into other team's codebases and haven't passed by their "adapter" types

I think I'll keep this as-is